Unity/ios/VonageUnityIntegration/ViewController.mm

```diff
@@ -34,7 +34,13 @@ bool compress(const std::unique_ptr<uint8_t[]>& inputArray,
                   uint32_t& outputSize){
         return Holographic::Compression::compress(inputArray, inputSize, outputArray, outputSize);
     }
-    
+
+    bool compress(CVPixelBufferRef depthDataMap,
```

Comment:
with the patch i sent you for webrtc we can remove the other function.

Reply:
Removed it.